test/avocado/seleniumlib.py

```diff
@@ -146,6 +151,9 @@ def get_debug_logs(self, logs=['browser','driver','client','server']):
         except WebDriverException as e:
             self.log.info("ERR: Unable to get logs: " + e.msg)
 
+    def execute(self, *args, **kwargs):
+        self.guest_machine.execute(*args, **kwargs)
+
```

Comment:
Where does this get used?

Reply:
I expect to use this inside selenium tests and it is better to have ti wrappered to have shorter notation, it helps to solve problmeof direct running command like: https://github.com/cockpit-project/cockpit/blob/master/test/avocado/selenium-base.py#L137 (what causes me troubles when GUEST is another machine than localhost) (for example when I test atomic container images in dowstream I have also there part that runs test against real atomic host on another machine with cockpit inside) what means that mentioned command is executed on another machine than expected